src/FakeItEasy.ruleset

```diff
@@ -11,6 +11,7 @@
     <Rule Id="SA1124" Action="Warning" />
     <Rule Id="SA1127" Action="None" />
     <Rule Id="SA1128" Action="None" />
+    <Rule Id="SA1413" Action="None" />

```

Comment:
The analyzer [prefers a trailing comma in multiline initializers](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1413.md). I do too, but there are many instances where we don't use one, and I wasn't sure of the prevailing sentiment. I'm happy to remove this and add the commas…

Reply:
I usually don't add a trailing comma, but I don't care if there is. Suppressing the warning is fine, as far as I'm concerned.